common/enablers/enableEpochsHandler.go

```diff
@@ -229,6 +235,530 @@ func (handler *enableEpochsHandler) RelayedNonceFixEnableEpoch() uint32 {
+	return epoch > handler.enableEpochsConfig.StakingV2EnableEpoch
+}
+
+// IsDoubleKeyProtectionFlagEnabledInEpoch returns true if DoubleKeyProtectionEnableEpoch is lower than the provided epoch
+func (handler *enableEpochsHandler) IsDoubleKeyProtectionFlagEnabledInEpoch(epoch uint32) bool {
+	return epoch >= handler.enableEpochsConfig.DoubleKeyProtectionEnableEpoch
+}
+
+// IsESDTFlagEnabledInEpoch returns true if ESDTEnableEpoch is lower than the provided epoch
+func (handler *enableEpochsHandler) IsESDTFlagEnabledInEpoch(epoch uint32) bool {
+	return epoch >= handler.enableEpochsConfig.ESDTEnableEpoch
+}
+
+// IsESDTFlagEnabledInSpecificEpochOnly returns true if ESDTEnableEpoch is the provided epoch
+func (handler *enableEpochsHandler) IsESDTFlagEnabledInSpecificEpochOnly(epoch uint32) bool {
+	return epoch == handler.enableEpochsConfig.ESDTEnableEpoch
+}
+
+// IsGovernanceFlagEnabledInEpoch returns true if GovernanceEnableEpoch is lower than the provided epoch
+func (handler *enableEpochsHandler) IsGovernanceFlagEnabledInEpoch(epoch uint32) bool {
+	return epoch >= handler.enableEpochsConfig.GovernanceEnableEpoch
+}
+
+// IsGovernanceFlagEnabledInSpecificEpochOnly returns true if GovernanceEnableEpoch is the provided epoch
+func (handler *enableEpochsHandler) IsGovernanceFlagEnabledInSpecificEpochOnly(epoch uint32) bool {
+	return epoch == handler.enableEpochsConfig.GovernanceEnableEpoch
+}
+
+// IsDelegationManagerFlagEnabledInEpoch returns true if DelegationManagerEnableEpoch is lower than the provided epoch
+func (handler *enableEpochsHandler) IsDelegationManagerFlagEnabledInEpoch(epoch uint32) bool {
+	return epoch >= handler.enableEpochsConfig.DelegationManagerEnableEpoch
+}
+
+// IsDelegationSmartContractFlagEnabledInEpoch returns true if DelegationSmartContractEnableEpoch is lower than the provided epoch
+func (handler *enableEpochsHandler) IsDelegationSmartContractFlagEnabledInEpoch(epoch uint32) bool {
+	return epoch >= handler.enableEpochsConfig.DelegationSmartContractEnableEpoch
+}
+
+// IsDelegationSmartContractFlagEnabledInSpecificEpochOnly returns true if DelegationSmartContractEnableEpoch is the provided epoch
+func (handler *enableEpochsHandler) IsDelegationSmartContractFlagEnabledInSpecificEpochOnly(epoch uint32) bool {
+	return epoch == handler.enableEpochsConfig.DelegationSmartContractEnableEpoch
+}
+
+// IsCorrectLastUnJailedFlagEnabledInEpoch returns true if correctLastUnJailedFlag is lower than the provided epoch
+func (handler *enableEpochsHandler) IsCorrectLastUnJailedFlagEnabledInEpoch(epoch uint32) bool {
+	return epoch >= handler.enableEpochsConfig.CorrectLastUnjailedEnableEpoch
+}
+
+// IsCorrectLastUnJailedFlagEnabledInSpecificEpochOnly returns true if correctLastUnJailedCurrentEpochFlag is the provided epoch
+func (handler *enableEpochsHandler) IsCorrectLastUnJailedFlagEnabledInSpecificEpochOnly(epoch uint32) bool {
+	return epoch == handler.enableEpochsConfig.CorrectLastUnjailedEnableEpoch
+}
```

Comment:
```suggestion
// IsCorrectLastUnJailedFlagEnabledInEpoch returns true if CorrectLastUnjailedEnableEpoch is lower than the provided epoch
func (handler *enableEpochsHandler) IsCorrectLastUnJailedFlagEnabledInEpoch(epoch uint32) bool {
	return epoch >= handler.enableEpochsConfig.CorrectLastUnjailedEnableEpoch
}

// IsCorrectLastUnJailedFlagEnabledInSpecificEpochOnly returns true if CorrectLastUnjailedEnableEpoch is the provided epoch
func (handler *enableEpochsHandler) IsCorrectLastUnJailedFlagEnabledInSpecificEpochOnly(epoch uint32) bool {
	return epoch == handler.enableEpochsConfig.CorrectLastUnjailedEnableEpoch
}
```
change comments to have the same pattern as the others?

Reply:
right, missed them